src/Voron/Data/PostingLists/PostingList.cs

```diff
@@ -131,12 +132,14 @@ private void ResizeCursorState()
             return state;
         }
 
+        [Conditional("DEBUG")]
         public void Verify()
```

Comment:
Is this correct? If we rely on calling Verify in tests this change would essentially nullify the test.

Reply:
We are never calling this right now.
This is debug code only, I used it to verify that the structure of the tree is sound.
We are testing the actual semantics by reading the data, which does the same better.
This is at a lower level.